apps/chat2/chat2.nim

```diff
@@ -212,7 +212,11 @@ proc publish(c: Chat, line: string) =
       # Attempt lightpush
       asyncSpawn c.node.lightpushPublish(some(DefaultPubsubTopic), message)
```

Comment:
Maybe would be good to apply the same error handling for lightpush as below for publis.
As an example app, it is good to have consistent. WDYT?

Reply:
Great idea! Makes sense to maintain consistency between both. I'll do it in a separate PR and merge to master once they are both ready :))

Reply:
Noticed that `lightpushPublish` already had error handling set up and when publishing Lightpush messages from REST it is already implemented what I did in this PR.

It was a matter of adapting the `chat2` and a small wrapper function over `lightpushPublish` that didn't return a result. So I did these small fixes and added them to this PR :)